components/tabcordion/src/overrides/panelBody.js

```diff
@@ -31,7 +31,7 @@ const PanelBody = ({ state: { mode, selected }, setPanelHeight, ...rest }) => {
 	});
 	const AnimatedBody = animated(Body);
 
-	return <AnimatedBody ref={measureRef} style={fade} {...rest} />;
+	return <Body ref={measureRef} {...rest} />;
 };
```

Comment:
Can we put `<AnimatedBody />` back?... you've removed the fade effect 😭

Reply:
ooops